testng-eclipse-plugin/src/main/org/testng/eclipse/ui/SuiteRunInfo.java

```diff
@@ -177,7 +177,7 @@ public long getRunDuration() {
   }
 
   public int getNewMax() {
-    return (m_methodTotalCount * m_testsTotalCount) / (m_testCount + 1);
+    return (m_methodTotalCount * m_testsTotalCount + 1) / (m_testCount + 1);
```

Comment:
this is to handle cases that `m_testsTotalCount=1`, to make sure the new max is at least 1 rather than 0.